test/built-ins/RegExp/prototype/Symbol.match/y-fail-return.js

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: Stops match execution after first match failure
```

Comment:
Similar to this test case, should we add another one making sure that the loop at **8. g.** stops at the first non-match when the **global** flag is present?